salt/pillar/netbox.py

```diff
@@ -1015,7 +1015,10 @@ def _get_site_prefixes(
     # Clean up duplicate data in the dictionary
     prefix_count = 0
     for prefix in site_prefixes_results:
-        del site_prefixes_results[prefix_count]["site"]
+        if "site" in site_prefixes_results[prefix_count].keys():
```

Comment:
For checking if an item is in a dictionary you just call in against the dictionary reference itself--you don't call in on `.keys()`: `if "site" in site_prefixes_results[prefix_count]:`. However you may just want to try to delete both:

```python
site_prefixes_results[prefix_count].pop("site", None)
site_prefixes_results[prefix_count].pop("scope", None)
```